e2e/tests-dfx/update_settings.bash

```diff
@@ -25,40 +73,40 @@ teardown() {
     ALICE_WALLET=$(dfx --identity alice identity get-wallet)
     BOB_WALLET=$(dfx --identity bob identity get-wallet)
 
-    dfx canister create hello
+    dfx canister --wallet "${ALICE_WALLET}" create hello
     dfx build hello
-    dfx canister install hello
+    dfx canister --wallet "${ALICE_WALLET}" install hello
```

Comment:
Let's leave this test as is with respect to these two ` canister create|install` lines, and if necessary, make a second test that creates/installs using `--wallet`.

Reply:
I already did - see https://github.com/dfinity/sdk/blob/1b0a74e4c714c51885c7fa911eab8a15ad3d09bf/e2e/tests-dfx/update_settings.bash#L17